changelog.d/392.doc

```diff
@@ -0,0 +1 @@
+Fix the documentation around the command line arguments for the emails migration script.
```

Comment:
```suggestion
Fix the documentation around the command line arguments for the email migration script.
```

While it does migrate multiple emails, "email migration script" sounds correct. I cannot explain to you why my English background is telling me that though.